test/irmin-mem/test_mem.ml

```diff
@@ -16,7 +16,8 @@
 
 open Lwt.Infix
 
-let store = Irmin_test.store (module Irmin_mem.Make) (module Irmin.Metadata.None)
+let store =
+  Irmin_test.store (module Irmin_mem.Make) (module Irmin.Metadata.None)
```

Comment:
Weird.

Reply:
The line was 81 characters long, I don't know exactly why it didn't break, but not having a space before the closing parenthesis makes it easier for Format to compute the size of the box and whether it should break (due to the fact that the break we use before the closing parenthesis is not a "vanilla" Format break but something more custom)